membership-attribute-service/test/repositories/MembershipAttributesSerializer.scala

```diff
@@ -1,5 +1,6 @@
 package repositories
```

Comment:
If nothing uses this then should we just delete it?

Reply:
The DynamoAttributeServiceTest uses it. I think we should keep that test.